provisioning/gu-membership-attribute-service-dist/upstart.conf

```diff
@@ -19,7 +19,7 @@ chdir /<APP>
 script
 TOTAL_MEMORY=$(grep MemTotal /proc/meminfo | awk '{ print $2 }')
 HEAP_SIZE_IN_MB=$(python -c "print int($TOTAL_MEMORY * 0.8 / 1024)")
-COMMAND="/<APP>/<APP>-1.0-SNAPSHOT/bin/<APP> -J-Xmx${HEAP_SIZE_IN_MB}m -J-Xms${HEAP_SIZE_IN_MB}m -Dplay.crypto.secret=<PLAY_APP_SECRET> -Dconfig.resource=<Stage>.conf"
+COMMAND="/<APP>/<APP>-1.0-SNAPSHOT/bin/<APP> -J-Xmx${HEAP_SIZE_IN_MB}m -J-Xms${HEAP_SIZE_IN_MB}m -Dplay.crypto.secret=<PLAY_APP_SECRET>"
 echo "$COMMAND" >/<APP>/cmd.txt
 $COMMAND >$LOGFILE 2>&1
 end script
```

Comment:
This script is downloaded from s3 in the launch configuration so we have to make sure to update it in the bucket before deploying these changes